### DISCUSSION TITLE
Reconsolidated folders for printing

### DISCUSSION THREAD

**Discussion by TheTimApple** [jrnl-org/jrnl#1842] [Ideas]
I like the idea of organizing by folders. But I wonder if there is a way to consolidate everything back into one file or a pdf or something for printing. It would be a pain to put it all back together manually. Thanks!

**Comment by micahellison** [comment]
Hi @Parzzix, no matter how your journal is stored, you can use jrnl's `--format` argument to display its contents in a variety of [formats](https://jrnl.sh/en/stable/formats/) and dump the results to a file.

For instance, to output your whole journal to a file called `journal.txt` in the `pretty` format, you can enter:
```
jrnl --format pretty > journal.txt
```
Then open that file in the program of your choice and print it or do whatever else you'd like with it.
